### ISSUE TITLE
Add endpoints for Job Information and Company Address

### ISSUE THREAD

**Issue by vinfante** [wizeline/ror-remotepayment-wizeline#78]
Create endpoints for the models:
JobInformation
CompanySite

- [ ] /api/v1/job_informations
- [ ] /api/v1/job_informations/:id
- [ ] /api/v1/company_sites
- [ ] /api/v1/company_sites/:id

For the endpoints above, the frontend will send:
- :id for any specific resource.

these endpoints result from declaring the routes like: 
```
  namespace :api do
    namespace :v1 do
      ...
      resources :job_informations
      resources :company_site
    end
  end
```

The access to JobInformation API through a Contractor user will limit the resulting information and actions to affect only the information that pertain to the given Contractor user, this is, Contractor A is not able to see or modify information from any other Contractor user.
The access to CompanySite API through a Contractor user will limit the resulting information to the active Company Site as read only.

The access to JobInformation APIs through a Finance Staff user will limit the actions to read only, this is, neither modifications nor additions/deletions to this data are to be done using said role. Information is going only to be retrieved.
The access to CompanySite API through a Finance Staff user will limit the resulting information to the active Company Site for display and modification of the active Company Site.

The user is identified by the required JWT token in the Authorization Bearer header.

| Resource | Action | Contractor | Finance | enpoint usage |
| --- | --- | --- | --- | --- |
| job_informations | GET index | limit to self | X | /api/v1/job_informations |
| | POST create | limit to self | forbidden | /api/v1/job_informations/:id |
| | GET show (id) | limit to self | X | /api/v1/job_informations/:id |
| | PATCH update (id) | limit to self | forbidden | /api/v1/job_informations/:id |
| | PUT update (id) | limit to self | forbidden | /api/v1/job_informations/:id |
| | DELETE delete (id) | forbidden | forbidden | /api/v1/job_informations/:id |

| Resource | Action | Contractor | Finance | enpoint usage |
| --- | --- | --- | --- | --- |
| company_sites | GET index | limit to active | limit to active | /api/v1/company_site |
| | POST create | forbidden | limit to active | /api/v1/company_site/:id |
| | GET show (id) | limit to active | limit to active | /api/v1/company_site/:id |
| | PATCH update (id) | forbidden | limit to active | /api/v1/company_site/:id |
| | PUT update (id) | forbidden | limit to active | /api/v1/company_site/:id |
| | DELETE delete (id) | forbidden | forbidden | /api/v1/company_site/:id |

**Comment by vinfante** [comment]
DEPENDS ON:
Closing PR #71

**Comment by JossEsparza** [comment]
This issue is duplicated.
Issue will be fixed with: #93 and #91
